app/models/publication.rb

```diff
@@ -155,7 +155,7 @@ def update_from_pubmed
 
   # @return [Boolean] true if .save is successful
   def rebuild_pub_hash
-    raise 'rebuilding WOS records is unsupported, unimplemented' if wos_uid
+    raise 'rebuilding WOS records is unsupported, unimplemented' if wos_pub?
```

Comment:
a record can have a provenance of pubmed, and still have a wos_uid added to it -- in this case, we can rebuild the pub_hash from the pubmed record if we want ... this is a better constraint check